src/sentry/event_manager.py

```diff
@@ -326,8 +326,6 @@ def normalize(self):
             except Exception:
                 # XXX: we should consider logging this.
                 pass
-            else:
-                data['tags'].extend(inst.iter_tags())
```

Comment:
assuming this was moved somewhere, but im not familiar with the code so just checking it was intentional